README.md

```diff
@@ -28,18 +28,27 @@ ideal domain is a UFD.
 ErDBeere wants to represent all the data in the answers above, i.e., *concrete
 examples* and *abstract implications*, in the nicest possible manner.
 
-## Installation
+## Local Installation
```

Comment:
>  If you do this for the first time, the database will still empty. In that case you can enter the container and seed the database:

Reseeding works perfectly fine. However, there are no instructions for how to setup an account or what development credential to use.